keyboards/treadstone48/rev1/info.json

```
@@ -0,0 +1,257 @@
+{
+  "keyboard_name": "Treadstone48",
+  "url": "https://github.com/marksard/Keyboards",
+  "maintainer": "marksard",
+  "width": 14.75,
+  "height": 4,
+  "layouts": {
+    "LAYOUT": {
```

The name of this key needs to match the layout macro with which it corresponds.

https://github.com/qmk/qmk_firmware/blob/f2c1aa69d18b70de6a4e23e5441d8096a61bc519/keyboards/treadstone48/rev1/rev1.h#L55

```suggestion
    "LAYOUT_base": {
```

`LAYOUT_full` and `LAYOUT_rs` both need to have similar keys in this file as well.